playwright-e2e/playwright.config.ts

```diff
@@ -16,8 +16,8 @@ const testConfig: PlaywrightTestConfig = {
   globalSetup: require.resolve('./fixtures/global-setup'),
   testDir: '.',
   testMatch: '**/*.spec.ts',
-  /* Maximum timeout per test. Test should be short and takes less than 2 min or 3 min in docker container to run */
-  timeout: process.env.CI ? 180 * 1000 : 120 * 1000,
+  /* Maximum timeout per test. Each test should be short and takes less than 3 min to run */
+  timeout: 180 * 1000,
```

Comment:
Set test max timeout same for CI and localhost, to 3 min. During my local testing, one test ran over 2 min and it failed due to timeout.